src/Setup/WixBA/WixBA.cs

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
 
+// WixBAFacelift TODO list:
+// Compile against NetFx452 (supported).  Moved to Net45 to check out using WindowChrome, then backed out Chrome.  check with wix-devs, then update Bundle when framework consenuse exist.
```

Comment:
Sorry I should have removed that comment.  I was just a reminder to have
further discussion pointing out that I tried to use the Chrome object, but
backed it out because we want to compile against NetFx40. I don't recall
why folks indicated that the Chrome object was a better path, but I don't
see any problem staying with NetFx40.

On Thu, Jul 28, 2016 at 3:09 AM, Rob Mensching <notifications@github.com>
wrote:

> In src/Setup/WixBA/WixBA.cs
> <https://github.com/wixtoolset/wix4/pull/208#discussion_r72580324>:
>
> > @@ -1,5 +1,7 @@
> >  // Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
> >
> > +// WixBAFacelift TODO list:
> > +// Compile against NetFx452 (supported).  Moved to Net45 to check out using WindowChrome, then backed out Chrome.  check with wix-devs, then update Bundle when framework consenuse exist.
>
> Do we have to? My hope is that if we find a NETFX v4.0 that we'll be happy
> to use that. Do we need v4.5+?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/208/files/5b08dba511d7438508767effab67896442e4ad1b#r72580324>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHyCLaCuslwW6U1ISG13rYxIp2oi3ugsks5qaGPLgaJpZM4JVm8P>
> .
>